Ryujinx/Ui/MainWindow.cs

```diff
@@ -0,0 +1,564 @@
+﻿using DiscordRPC;
+using Gtk;
+using GUI = Gtk.Builder.ObjectAttribute;
+using Ryujinx.Audio;
+using Ryujinx.Common.Logging;
+using Ryujinx.Graphics.Gal;
+using Ryujinx.Graphics.Gal.OpenGL;
+using Ryujinx.HLE.FileSystem;
+using Ryujinx.Profiler;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+
+namespace Ryujinx.UI
+{
+    public class MainWindow : Window
+    {
+        public static bool DiscordIntegrationEnabled { get; set; }
+
+        public static DiscordRpcClient DiscordClient;
+
+        public static RichPresence DiscordPresence;
+
+        private static IGalRenderer _renderer;
+
+        private static IAalOutput _audioOut;
+
+        internal static HLE.Switch _device;
+
+        private static Application _gtkapp;
+
+        private static ListStore _tableStore;
+
+        private static bool _gameLoaded = false;
+
+#pragma warning disable 649
+        [GUI] Window         _mainWin;
```

Comment:
```suggestion
        [GUI] Window        _mainWin;
```
There is an extra empty space in all those `[GUI]` lines.